multiraft/multiraft.go

```diff
@@ -658,8 +679,9 @@ func (s *state) handleWriteResponse(response *writeResponse, readyGroups map[uin
 				if entry.Data != nil {
 					var command []byte
 					commandID, command = decodeCommand(entry.Data)
-					s.sendEvent(&EventCommandCommitted{commandID, command})
+					s.sendEvent(&EventCommandCommitted{groupID, commandID, command})
```

Comment:
Might want to set this composite literal to use more explicit initialization style.